api/repositories/build_repository_test.go

```diff
@@ -585,7 +603,7 @@ var _ = Describe("BuildRepository", func() {
 		})
 
 		It("returns an empty array (as no roles assigned)", func() {
-			buildRecords, fetchError = buildRepo.ListBuilds(ctx, authInfo)
+			buildRecords, fetchError = buildRepo.ListBuilds(ctx, authInfo, listMessage)
```

Comment:
I would suggest pushing `buildRepo.ListBuilds` invocation to `Describe("ListBuilds"` -> `JustBeforeEach`. Thus invoking the repo on every `It` would not be necessary